src/Model/RefundTypeInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Model;
+
+interface RefundTypeInterface
+{
+    public const ORDER_ITEM_UNIT = 'order_item_unit';
+
+    public const SHIPMENT = 'shipment';
+
+    public static function orderItemUnit(): self;
+
+    public static function shipment(): self;
+
+    /**
+     * @psalm-param mixed $variable
+     */
+    public function equals($variable = null): bool;
+
+    /**
+     * @psalm-param mixed $value
+     */
+    public static function isValid($value): bool;
```

Comment:
😱 static methods in interfaces 😱 I didn't even know we can do that 😄 Nevertheless, it's definitely not proper coding, static methods are meant to be called without created object, but they need a function body for that 💃 Can we change them to be not static? I know they're done this way in the `Enum` class.

Reply:
Is not possible to call static method from parent in any not static method

Reply:
Indeed, but it does not mean we should go with such an abomination as static methods in the interface 😄 
What if we add checking `Enum` instance in `RefundEnumType::converToPHPValue` method and remove this functions from both `RefundTypeInterface` and `RefundType`?

```
if (
    $value instanceof RefundTypeInterface &&
    $value instanceof Enum &&
    !$value::isValid($value)
) {
    // ...
}
```

This way we're sure that the object implements `RefundTypeInterface` (which is required), is an enum (which is also required) and is a valid enum 🚀 

cc @pamil as you were also a part of this discussion

Reply:
Its my first version, before I added this methods. For me both are good

Reply:
As far as I remember, the first version was `$value instanceof RefundType && !$value::isValid($value)`, which was not using the interface - that was a problem 🚀